includes/REST/ProductController.php

```diff
@@ -103,7 +103,7 @@ public function register_routes() {
                     'args'                => [
                         'force' => [
                             'type'        => 'boolean',
-                            'default'     => false,
+                            'default'     => true,
```

Comment:
do not make this value default to true, leave this as false

Reply:
This will break backward compatibility as the current value is not remove.
Are you sure?

Reply:
the default value should be false as it was before, if you want to force delete, send this value as true from your API request, set the default value to false as it was before.

Reply:
We don't have to worry about backward compatibility, because only places we used force value with Product API.

Reply:
Yes! My mistake: I was advocating about keeping the compatibility but wrote the opposite...
Fixed!